core/src/main/java/apoc/warmup/Warmup.java

```diff
@@ -62,6 +62,7 @@ private String subPath(File file, String fromParent) {
         return sb.toString();
     }
 
+    @Deprecated
```

Comment:
It would be nice to add a note about why it is being deprecated. There is a `deprecatedBy` field in the `@Procedure` annotation.